packages/react-components/src/echarts/extensions/minMaxDataStreamValues/updateDataStreamMinMax.ts

```diff
@@ -0,0 +1,37 @@
+import { EChartsExtensionInstallRegisters } from 'echarts/types/src/extension';
+import { findMinMax } from './minMaxDataStreamSync';
+import useDataStore from '../../../store';
+
+// Echarts core use type does not map correctly to the echarts extension type so exporting as any
+// eslint-disable-next-line
+export const dataStreamMinMaxSyncExtension: any = (registers: EChartsExtensionInstallRegisters) => {
+  registers.registerUpdateLifecycle(
+    'series:afterupdate',
+    (_ecModel, _api, params) => {
```

Comment:
How often is this function called? should we think about throttling it since it does iterator all of the data points in every series?